archinstall/default_profiles/desktops/plasma.py

```diff
@@ -0,0 +1,48 @@
+from typing import List, Optional, Any, TYPE_CHECKING
+
+from archinstall.default_profiles.profile import ProfileType, GreeterType
+from archinstall.default_profiles.xorg import XorgProfile
+
+if TYPE_CHECKING:
+	_: Any
+
+class PlasmaProfile(XorgProfile):
+	def __init__(self):
+		super().__init__('KDE Plasma', ProfileType.DesktopEnv, description='')
+
+	@property
+	def packages(self) -> List[str]:
+		return [
+			"plasma-meta",
+			"konsole",
+			"kwrite",
+			"dolphin",
+			"ark",
+			"plasma-workspace",
+			"egl-wayland"
+		]
+
+	@property
+	def default_greeter_type(self) -> Optional[GreeterType]:
+		return GreeterType.Sddm
+
+# 2024-04-16 TODO deprecated Class with old naming, remove in a future version
+class KdeProfile(XorgProfile):
```

Comment:
With backards compability I didn't mean to leave the old profile configuration around but to make sure that reading configurations with the old name will be parsed correctly. Leaving this config around will display 2 profiles now. 

The backwards compability sits in 

    def parse_profile_config(self, profile_config: Dict[str, Any]) -> Optional[Profile]:
        """
        Deserialize JSON configuration for profile
        """
        profile: Optional[Profile] = None


        # the order of these is important, we want to
        # load all the default_profiles from url and custom
        # so that we can then apply whatever was specified
        # in the main/detail sections
        if url_path := profile_config.get('path', None):
            self._url_path = url_path
            local_path = Path(url_path)


            if local_path.is_file():
                profiles = self._process_profile_file(local_path)
                self.remove_custom_profiles(profiles)
                self.add_custom_profiles(profiles)
            else:
                self._import_profile_from_url(url_path)


![image](https://github.com/archlinux/archinstall/assets/19396770/0f0ee86a-1bac-43c5-aed7-c97b60097c34)